app/backend/src/couchers/servicers/threads.py

```diff
@@ -133,16 +133,20 @@ def generate_reply_notifications(payload: jobs_pb2.GenerateReplyNotificationsPay
             reply = session.execute(select(Reply).where(Reply.id == database_id)).scalar_one()
             # the comment we're replying to
             parent_comment = session.execute(select(Comment).where(Comment.id == reply.comment_id)).scalar_one()
+            thread_replies_author_user_ids = (
+                session.execute(select(Reply.author_user_id).where(Reply.comment_id == parent_comment.id))
```

Comment:
Use `.where_users_column_visible(context, Reply.author_user_id)` here; and see next comment.